fuels-abigen-macro/tests/test_projects/token-ops/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+authors = ["Fuel Labs"]
+edition = "2021"
+license = "Apache-2.0"
+name = "token-ops"
+version = "0.1.0"
+
+[dependencies]
+fuel-gql-client = { version = "0.2", default-features = false }
+fuel-tx = "0.3"
+fuels-abigen-macro = "0.3"
+fuels-contract = "0.3"
+fuels-core = "0.3"
```

Comment:
Is there a reason not use a local path for these? Now you're testing against a pinned version, which isn't testing upgrades.

Reply:
Nope, no reason. It's just the output from `forc init` @ `0.6`.

Reply:
And we don't use any of these `fuels-*` because we're not creating tests under that forc project, just using the compiled contract + ABI.

Reply:
Ah I see. Then maybe to avoid headaches they should just be removed if they're not used?